lib/all-names.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/**
+  The purpose of this module is to collect all possible names and
+  to convert them to different formats.
+*/
+
+const makeKey = require('./make-key');
+
+// Collect names from:
+// 1) Rosters
+// 2) Matches
+// 3) playerinfo.json
+// Anywhere else???
+
+const nameLookup = {};
+
+const { teams } = require('../model/seasons').get();
+Object.keys(teams)
+  .map(tk => teams[tk])
+  .forEach(team => team.roster.forEach(p => {
+     nameLookup[makeKey(p.name)] = p.name;
+  }));
+
+// const matches = require('../model/matches').all();
```

Comment:
This might not be necessary after all, because the main place where it was a problem was in stats, which are created from match data, and include the names from the matches there.